rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -38,7 +41,7 @@ object SystemProcesses {
         dispatcher: Dispatch[Task, ListParWithRandomAndPhlos, TaggedContinuation]
     ): Task[Unit] =
       res.fold(err => Task.raiseError(OutOfPhlogistonsError), _.fold(Task.unit) {
-        case (cont, channels) => _dispatch(dispatcher)(cont, channels)
+        case (cont, channels) => Task.pure(_dispatch(dispatcher)(cont, channels).unsafeRunSync)
```

Comment:
This seems bad. You are making a Task that runs a Task unsafely... Why not just leave it the way it was?

Reply:
Yeah so this is what I wanted to show you. The test doesn't work unless I do this...

Reply:
Specifically it doesn't print the timestamp to stdout

Reply:
But as you say, it is ugly. I'm not sure how to fix.

Reply:
Hmm, interesting. I would expect that `Task.pure(t.unsafeRunSync) == t` would be a law about how `Task` behaves. I can look into this more closely to see if I can figure out what is going on.